### PR TITLE
config: AARCH64 config for verification

### PR DESCRIPTION
**Pull request by lsf37**
For ongoing verification of seL4 on AArch64. This config has SMMU and hypervisor support enabled. AArch64 also implies FPU support is on. The target board for now is the TX2, which has an SMMU.

When this is merged, I'm planning to add the config to preprocess and compile tests.

### CODE REVIEW

**Comment by lsf37** [comment]
Style test fails on the `.cmake` file being executable, which in this case we do want to keep (in analogy with the other configs).

**Comment by Xaphiosis** [comment]
Fine by me, but maybe someone other than us should review as well before merging?

**Review by Xaphiosis — APPROVED** [review]

File: configs/AARCH64_verified.cmake
Comment (kent-mcleod) [inline configs/AARCH64_verified.cmake:29]: Are you able to remove these lines? KernelRootCNodeSizeBits seems like an application specific increase and KernelMaxNumBootinfoUntypedCap isn't an option yet the one on line 30 is.

Comment (lsf37) [reply configs/AARCH64_verified.cmake:29]: yes, happy to

Comment (lsf37) [reply configs/AARCH64_verified.cmake:29]: we should probably check the other verified configs, where I copy/pasted these from.

**Comment by lsf37** [comment]
Removed the two lines for `KernelMaxNumBootinfoUntypedCap` and `KernelRootCNodeSizeBits`, use defaults instead.

**Pull request merged**
